terraform/locals.aws.tf

```diff
@@ -49,6 +49,98 @@ locals {
     try(vpc.region, "") == "us-east-2"
   }
 
+  # Network -> Security Groups
+  security_groups = [
+    {
+      name = "core"
+      description : "Security group for core node"
+      security_groups = [
+        {
+          name            = "bp-from-relay-${var.cardano_port}"
+          from_port       = var.cardano_port
+          to_port         = var.cardano_port
+          protocol        = "tcp"
+          description     = "Cardano port from relay"
+          ingress_sg_name = "relay"
+        },
+        {
+          name            = "bp-from-relay-ssh"
+          from_port       = 22
+          to_port         = 22
+          protocol        = "tcp"
+          description     = "SSH from relay"
+          ingress_sg_name = "relay"
+        },
+        {
+          name        = "bp-egress-anywhere"
+          egress      = true
+          rule        = "all-all"
+          description = "all traffic egress to anywhere"
+        }
+      ]
+    },
+    {
+      name        = "relay"
```

Comment:
Rename this to simply "node" or "cardano-node"

Reply:
Changed